Sources/MapboxNavigation/NavigationMapView+RoadNameLabeling.swift

```diff
@@ -1,6 +1,9 @@
 import MapboxMaps
 import MapboxDirections
 import MapboxCoreNavigation
+import os.log
+
+private let logger: OSLog = .init(subsystem: "com.mapbox.navigation", category: "RoadNameLabeling")
```

Comment:
Should we use private global constants in other classes too?

Reply:
We needed one here since you can't create stored properties in extensions. For consistency, we should either move constants to be global or use a manager for these OSLog constants